packages/firebase-frameworks/src/sveltekit/index.ts

```diff
@@ -26,8 +26,11 @@ export const handle = async (req: Request, res: Response) => {
     return res.writeHead(404, "Not Found").end();
   }
 
-  const body = await rendered.text();
-  return res.writeHead(rendered.status, Object.fromEntries(rendered.headers)).end(body);
+  const body = (await rendered.arrayBuffer()) as ArrayBuffer;
```

Comment:
nit, can we simply to `const body: ArrayBuffer = await rendered.arrayBuffer();`

Reply:
done, although the types coming from `rendered` are all `any`